docs/config/README.md

```diff
@@ -3010,7 +3011,8 @@ If you have an interesting example not covered there, feel free to share it ther
 | `symbol`      | `""`                            | The symbol used before displaying the command output.                                                                      |
 | `style`       | `"bold green"`                  | The style for the module.                                                                                                  |
 | `format`      | `"[$symbol($output )]($style)"` | The format for the module.                                                                                                 |
-| `disabled`    | `false`                         | Disables this `custom` module.                                                                                             |
+| `disabled`    | `false`                         | Disables this `custom` module.                      |
+| `os`          |                                 | Operating System name on which the module will be shown. |
```

Comment:
I think it might be worth adding some examples of the possible values here since it is not obvious what a user should use on a Mac e.g osx, macOS or darwin are all reasonable guesses.

Reply:
```
(linux, macos, windows, ... ) [See possible values](https://doc.rust-lang.org/std/env/consts/constant.OS.html)
```
Should this be enough in the options description ?